pages/platform/ai/training_tuto_02_build_custom_image/guide.en-gb.md

```diff
@@ -4,116 +4,293 @@ slug: training/build-use-custom-image
 
 ``` {.console}
-RUN apt-get update && apt-get install -y vim
+WORKDIR /workspace
+ENV HOME=/workspace
 ```
 
-You can copy files from your local directory inside docker image with the `COPY` prefix.
+> [!warning]
+>
+>*Declaring the `/workspace` directory as the `HOME` environment variable is necessary to ensure compatibility with AI Training.*
+>
+
+### Add job's files to the home directory
+
+Once the `/workspace` directory is set up, we can add our files (Python scripts, requirements.txt, and the Dockerfile) to it thanks to the `ADD` instruction. 
+
+Here is an example to add one specific file, like a `main.py` Python file:
+
+``` {.console}
+ADD main.py /workspace/
+```
 
-Example if you want to add the file `example.txt` at the root of the image :
+If our project is composed of many files, we can also add all our project's directory files by using:
 
 ``` {.console}
-COPY example.txt /example.txt
+ADD . /workspace
 ```
 
 > [!warning]
 >
-> Images in AI Training are not run as root user, but by an "ovh" user with UID 42420. It means that if you want to be able to write in a specific directory at runtime you will have to give it specific rights.
-> You can do it with the following instruction :
+>This command could allow you to load your dataset within your Docker image. However, this would increase its size considerably. A best practice is to put data outside, such as [Object Storage](https://www.ovhcloud.com/en-gb/public-cloud/object-storage/), and link it to the AI Training job during its launch.
 >
->     RUN chown -R 42420:42420 <your-target-directory>
 
-> [!warning]
+### Give the OVHcloud user access to the home directory
+
+Images in AI Training are not run as root user, but by an "OVHcloud" user with UID 42420. 
+
+It means that if we want to be able to create and write in a specific directory at runtime, **we will have to give it specific rights**. We can do it with the following instruction:
+
+``` {.console}
+RUN chown -R 42420:42420 <specific-directory>
+```
+
+As mentioned, the home directory for the "OVHcloud" user (with UID 42420) will be `/workspace`. We can then replace `<specific-directory>` by `/workspace`. But keep in mind that you can change the ownership of any other useful directory.
+
+**To summarize**, this is what our `Dockerfile` looks like for now:
+
+``` {.console}
+FROM tensorflow/tensorflow:latest-gpu
+
+WORKDIR /workspace
+ENV HOME=/workspace
+ADD . /workspace
+
+RUN chown -R 42420:42420 /workspace
+```
+
+### Install what we need
+
+As explained before, it is possible that the chosen Docker image does not include all the libraries that our project needs to work properly. In this case, we need to install them to ensure that our AI Training job works correctly.
+
+We distinguish two types:
+- Linux packages
+- Python libraries
+
+These installation instructions begin with `RUN` prefix, which is used to execute a command during the image building process.
+
+**Install system packages**
```

Comment:
For linux packages, if they are numerous, the user can create a `packages.txt` file and install it like that: 
`RUN xargs -a packages.txt apt-get install --yes`